code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -613,6 +613,28 @@
 		M.emote("shiver")
 	holder.remove_reagent("capsaicin", 5)
 
+/datum/reagent/frostoil/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed) // Eating frostoil now acts like capsaicin. Wee!
+	if(alien == IS_DIONA)
+		return
+	if(alien == IS_ALRAUNE) // VOREStation Edit: It wouldn't affect plants that much.
+		if(prob(5))
+			to_chat(M, "<span class='rose'>You feel a chilly, tingling sensation in your mouth.</span>")
+		M.bodytemperature -= rand(10, 25)
+		return
```

Comment:
"VOREStation edit" in Polaris code would be confusing.  If I try to compile this locally, I get "code\modules\reagents\Chemistry-Reagents\Chemistry-Reagents-Food-Drinks.dm:619:error: IS_ALRAUNE: undefined var".  Also, Travis seems to be completely missing from all Polaris pull requests.

Reply:
My bad, I left in the Alraune blurb from VS. also, yeah, Travis is 100% fuckin' gone and I have no idea why. @Atermonera

Reply:
Well, now that travis has been sufficiently appeased, it is, as expected, angry that this doesn't compile because of the IS_ALRAUNE define.

Reply:
Done, should be good for merge.